content/docs/using-regolith/install.md

```diff
@@ -15,6 +15,41 @@ If you're upgrading from an earlier version of Regolith, you may be interested i
 ### Ubuntu
 
 {{< tabs "ubuntu-tabs" >}}
+{{< tab "Ubuntu 24.10" >}}
+
+Regolith can be installed as system packages.  This makes updating and removing easier and more consistent.  The following steps describe how
+to configure your system to read packages from the Regolith package repository and install the desktop package.
+
+1. Register the Regolith public key to your local `apt`:
+
+   ```console
+   wget -qO - http://archive.regolith-desktop.com/regolith.key | \
+   gpg --dearmor | sudo tee /usr/share/keyrings/regolith-archive-keyring.gpg > /dev/null
+   ```
+
+1. Add the repository URL to your local `apt`:
+
+   ```console
+   echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
```

Comment:
question: the quote here doesn't get emitted to the file right?  I found that apt on jammy does not like this subset of the apt line quoted FWIW

Reply:
It does not. But that's a good point. Maybe we can just rewrite this to be a single line without `\` line break? Not sure if it's going to help readability though.

Reply:
Oh, sorry I notice this is what we already have on the site as is.  I guess my issue was due to copying from the markup rather than the rendered HTML so a non-issue.